src/gui/runtime/plugin/nvim_gui_shim.vim

```diff
@@ -99,6 +99,11 @@ command! -nargs=1 GuiPopupmenu call s:GuiPopupmenu(<args>)
 " GuiDrop('file1', 'file2', ...) is similar to :drop file1 file2 ...
 " but it calls fnameescape() over all arguments
 function GuiDrop(...)
+	" Check for user defined handler function
+	if exists('*GuiDropCustom')
+		return call("GuiDropCustom", a:000)
```

Comment:
Looks good. My only only nitpick would be the name of `GuiDropCustom` vs something like `GuiDropCustomHandler`

Reply:
Sure, I was struggling with a good name, I like `GuiDropCustomHandler` more.